great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -264,30 +267,24 @@ def __init__(  # noqa: C901 - 17
         concurrency: Optional[ConcurrencyConfig] = None,
         **kwargs,  # These will be passed as optional parameters to the SQLAlchemy engine, **not** the ExecutionEngine
     ) -> None:
-        """Builds a SqlAlchemyExecutionEngine, using a provided connection string/url/engine/credentials to access the
-        desired database. Also initializes the dialect to be used and configures usage statistics.
+        """Builds a SqlAlchemyExecutionEngine, using a provided connection string/url/engine/credentials to access the desired database.
+
+        Also initializes the dialect to be used and configures usage statistics.
```

Comment:
I think we configure usage statistics at the data context level.
```suggestion
        Also initializes the dialect to be used.
```